assets/js/blocks/product-best-sellers/index.js

```diff
@@ -29,6 +31,18 @@ registerBlockType( 'woocommerce/product-best-sellers', {
 	attributes: {
 		...sharedAttributes,
 	},
+	transforms: {
+		from: [
+			{
+				type: 'block',
+				blocks: difference( rowColumnBlockTypes, [ 'woocommerce/product-best-sellers' ] ),
```

Comment:
If you use `without` instead of `difference`, this doesn't need to be an array (i also benchmarked it out of curiosity, and it seems `without` is slightly more performant, too)